Examples/S3EventNotifier/README.md

```diff
@@ -30,10 +30,14 @@ If there are no errors, a ZIP file should be ready to deploy, located at `.build
 
 ## Deploy
 
+[!IMPORTANT]
+The Lambda function and the S3 bucket must be located in the same AWS Region. In the code below, we use `eu-west-1` (Ireland). 
```

Comment:
I think GH markdown requires this to be in block quotes, so
```
> [!IMPORTANT]
> The Lambda function...
```

Reply:
It's a GitHub specific tag https://github.com/orgs/community/discussions/16925

Reply:
You're correct - thank you